actors/miner/tests/util.rs

```diff
@@ -204,8 +261,1024 @@ impl ActorHarness {
+                rt,
+                &pc,
+                self.make_prove_commit_params(pc.info.sector_number),
+                ProveCommitConfig::empty(),
+            );
+            info.push(sector);
+        }
+        rt.reset();
+        info
+    }
+
+    pub fn get_deadline_info(&self, rt: &MockRuntime) -> DeadlineInfo {
+        let state = self.get_state(rt);
+        state.recorded_deadline_info(&rt.policy, rt.epoch)
+    }
+
+    fn make_pre_commit_params(
+        &self,
+        sector_no: u64,
+        challenge: ChainEpoch,
+        expiration: ChainEpoch,
+        sector_deal_ids: Vec<DealID>,
+    ) -> PreCommitSectorParams {
+        PreCommitSectorParams {
+            seal_proof: self.seal_proof_type,
+            sector_number: sector_no,
+            sealed_cid: make_sealed_cid(b"commr"),
+            seal_rand_epoch: challenge,
+            deal_ids: sector_deal_ids,
+            expiration: expiration,
+            // unused
+            replace_capacity: false,
+            replace_sector_deadline: 0,
+            replace_sector_partition: 0,
+            replace_sector_number: 0,
+        }
+    }
+
+    fn make_prove_commit_params(&self, sector_no: u64) -> ProveCommitSectorParams {
+        ProveCommitSectorParams { sector_number: sector_no, proof: vec![0u8; 192] }
+    }
+
+    fn pre_commit_sector(
+        &self,
+        rt: &mut MockRuntime,
+        params: PreCommitSectorParams,
+        conf: PreCommitConfig,
+        first: bool,
+    ) -> SectorPreCommitOnChainInfo {
+        rt.set_caller(*ACCOUNT_ACTOR_CODE_ID, self.worker);
+        rt.expect_validate_caller_addr(self.caller_addrs());
+        self.expect_query_network_info(rt);
+
+        if params.deal_ids.len() > 0 {
+            let vdparams = VerifyDealsForActivationParams {
+                sectors: vec![SectorDeals {
+                    sector_expiry: params.expiration,
+                    deal_ids: params.deal_ids.clone(),
+                }],
+            };
+            let vdreturn = VerifyDealsForActivationReturn {
+                sectors: vec![SectorWeights {
+                    deal_space: conf.deal_space.unwrap() as u64,
+                    deal_weight: conf.deal_weight,
+                    verified_deal_weight: conf.verified_deal_weight,
+                }],
+            };
+
+            rt.expect_send(
+                *STORAGE_MARKET_ACTOR_ADDR,
+                MarketMethod::VerifyDealsForActivation as u64,
+                RawBytes::serialize(vdparams).unwrap(),
+                TokenAmount::from(0),
+                RawBytes::serialize(vdreturn).unwrap(),
+                ExitCode::Ok,
+            );
+        }
+        // in the original test the else branch does some redundant checks which we can omit.
+
+        let state = self.get_state(rt);
+        if state.fee_debt > TokenAmount::from(0) {
+            rt.expect_send(
+                *STORAGE_POWER_ACTOR_ADDR,
+                PowerMethod::EnrollCronEvent as u64,
```

Comment:
This is a copy paste error from the call below.  In this case you expect a send to the burnt funds actor.

Reply:
ooooops! Will fix.

Reply:
done.